php/Terminus/Models/Environment.php

```diff
@@ -876,6 +869,63 @@ public function workflow($workflow) {
   }
 
   /**
+   * Sifts through the backups data and adds needful properties
+   *
+   * @param [stdClass] $raw_backups Backups data object from Request data
+   * @return [array] $backups
+   */
+  private function annotateBackups($raw_backups) {
```

Comment:
I admit that moving to model/collection for backups might be a substantial amount of work :-(